content/root/best-practices-guide.textile

```diff
@@ -0,0 +1,97 @@
+---
+title: Best Practices Guide
+section: root
+index: 30
+jump_to:
+  Contents:
+    - Ably 101
+    - Using Ably like a pro
+---
+
+h2(#title). Ably Best Practices Guide
+
+In order to avail the best advantage of our platform, be sure to follow this best practices guide while building your applications using Ably.
```

Comment:
I don't like this intro I am afraid.  I think it would be better to explain that we've created this handy best practice guide in two sections:
- "Ably 101 - key concepts" - before you get started, make sure you understand 
- "Ably implementation checklist" - once you've started implementing Ably, we've compiled a quick checklist to help you avoid common mistakes or misunderstandings we see when developers implement Ably.